hack/parallel.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+oc get nodes > /dev/null 2>&1
+if [ $? -ne 0 ]; then
+    echo "Cluster not found, check KUBECONFIG"
+    exit 1
+fi
+
+oc get ns kubevirt > /dev/null 2>&1
```

Comment:
@oshoval Kubevirt is not always being installed on a "kubevirt" NS. You can use KUBEVIRT_ISNTALLED_NAMESPACE env var to derive the NS, otherwise use the "kubevirt" NS as default.

Reply:
done, thanks